packages/cache/src/internal/cacheUtils.ts

```diff
@@ -113,3 +113,28 @@ export async function isGnuTarInstalled(): Promise<boolean> {
   const versionOutput = await getVersion('tar')
   return versionOutput.toLowerCase().includes('gnu tar')
 }
+
+export async function getAzCopyCommand(): Promise<string | undefined> {
+  // Always prefer the azcopy10 alias first, which is the correct version on Ubuntu.
```

Comment:
What do you mean by correct version? What happens if we just do the version 10 or newer check?

Reply:
`azcopy` on Ubuntu is version 7.x.x, which uses a completely different CLI.  So we need to use the `azcopy10` alias on Ubuntu to get to version 10.

This is different on a Mac, where `azcopy` is v10.  Windows currently does not have AzCopy installed, but I already filed an issue to fix that.

I'm not sure why it's setup this way, but this logic is needed to ensure we get the correct version on Ubuntu.